test/solidity/oz/erc20votes/erc20Votes.js

```diff
@@ -0,0 +1,121 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+const { Contract } = require('../../../constants')
+
+const AMOUNT_TO_MINT = 100
+const sleep = (timeToSleep) => new Promise((r) => setTimeout(r, timeToSleep))
+const FUTURE_LOOKUP_ERROR = 'ERC5805FutureLookup'
+
+describe('@solidityequiv3 Solidity OZ ERC-20 Votes', function () {
+    let contract, wallet, wallet2
+    const TIME_INCREMENT = 10000
+
+    before(async function () {
+        const signers = await ethers.getSigners()
+        wallet = signers[0]
+        wallet2 = signers[1]
+        const votesFactory = await ethers.getContractFactory(Contract.OZ.ERC20VotesTest)
+        contract = await votesFactory.deploy(AMOUNT_TO_MINT, {
+            gasLimit: 8000000,
+        })
+    })
+
+    it('should should deploy contract', async function () {
+        const deployed = await contract.deployed()
+        expect(deployed).to.exist
+    })
+
+    it('should create/mint the erc20 tokens', async function () {
```

Comment:
Q: is this test doing the intended logic? Doesn't seem to match the description

Reply:
The minting is done in the Smart Contracts constructor. 
I am just checking if we have the tokens.

Reply:
The description should match the logic then

Reply:
But then you should check if it's a duplicate of the next test